tests/suite/advanced_routing_test.go

```diff
@@ -116,17 +116,25 @@ func expectRequestToRespondFromExpectedServer(
 	appURL, address, expServerName string,
 	headers, queryParams map[string]string,
 ) error {
-	status, body, err := framework.Get(appURL, address, timeoutConfig.RequestTimeout, headers, queryParams)
+	GinkgoWriter.Printf("Expecting request to respond from the server %q\n", expServerName)
+	status, body, err := framework.Get(appURL, address, timeoutConfig.RequestTimeout, headers, queryParams, true)
 	if err != nil {
+		GinkgoWriter.Printf("ERROR making GET request to %q: %v\n", appURL, err)
```

Comment:
I don't think we need to print this error again. It will already have been logged in the `framework.Get` function.